tfx/experimental/pipeline_testing/examples/chicago_taxi_pipeline/taxi_pipeline_regression_e2e_test.py

```diff
@@ -113,7 +113,7 @@ def testTaxiPipelineBeam(self):
         if component.id != model_resolver_id
     ]
 
-    stub_launcher = stub_component_launcher.get_stub_launcher_class(
+    stub_launcher = StubComponentLauncher.get_stub_launcher_class(
```

Comment:
How about creating a new class by inheriting StubComponentLauncher here, too? I think that it would be better to have consistent API style between beam and KFP.

Reply:
It looks like that there is a gap between the usage in beam_dag_runner and kubeflow_dag_runner. It might be confusing to users who are not familiar with python module path resolution.

My two cents is it would be better to move get_stub_launcher_class() call to below 'StubComponentLauncher` class. One problem is that the name of the function `get_stub_launcher_class` doesn't match with its behavior then, because we don't really use the return value of this function. So possibly we might need to rename it, or I still think that decorator-based API would be cleaner.